src/pretix/control/views/event.py

```diff
@@ -399,6 +403,56 @@ def post(self, request, *args, **kwargs):
             return self.get(request)
 
 
+class MailSettingsPreview(EventPermissionRequiredMixin, View):
+    permission = 'can_change_settings'
+
+    # return the origin text if key is missing in dict
+    class SafeDict(dict):
+        def __missing__(self, key):
+            return '{' + key + '}'
+
+    @cached_property
+    def items(self):
+        return ['mail_text_order_placed', 'mail_text_order_paid', 'mail_text_order_free', 'mail_text_resend_link',
+                'mail_text_resend_all_links', 'mail_text_order_changed', 'mail_days_order_expire_warning',
+                'mail_text_order_expire_warning', 'mail_text_waiting_list']
+
+    @cached_property
+    def dummy_data(self):
+        return {
+            'mail_text_order_placed': {
+                'event': str(self.request.event.name),
```

Comment:
You can fix that by using new-style formats here as well:

```
'invoice_name': _('Invoice for {}.').format(self.request.event.name)
```

They can deal with the internationalized strings. In the long run, we eventually want to use ``.format()`` everywhere.